packages/react-navigation-native/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@use-funnel/react-navigation-native",
-  "version": "0.0.15",
+  "version": "0.0.16",
```

Comment:
Don't fix version directly. revert and execute command `pnpm changeset` at root, and select patch version.

Reply:
@minuukang I didn't know much about changeset, so I didn't understand how to update version.
As you suggested, I select the patch version using changeset instead of modifying version directly!
please check ! Thank you